src/main/java/io/confluent/connect/elasticsearch/ElasticsearchSinkConnectorConfig.java

```diff
@@ -51,6 +53,12 @@ protected static ConfigDef baseConfigDef() {
           .define(CONNECTION_URL_CONFIG, Type.LIST, Importance.HIGH,
                   "List of Elasticsearch HTTP connection URLs e.g. ``http://eshost1:9200,http://eshost2:9200``.",
                   group, ++order, Width.LONG, "Connection URLs")
+          .define(CONNECTION_USERNAME, Type.STRING, Importance.LOW,
+                  "The username to use for HTTP Basic Authentication.",
+                  group, ++order, Width.LONG, "Connection Username")
+          .define(CONNECTION_PASSWORD, Type.STRING, Importance.LOW,
```

Comment:
Could we change the Type here to `Type.PASSWORD`? This will mask the output when the value is logged so passwords aren't accidentally leaked via logs.